stelter/StelterAPI
# Stelter.js - hide excluded getting started options

1. Update for stelter.js to hide getting started options if the only corresponding suggestions are exclusions. Also getting started now loads with fetch instead of xhttp request since we have polyfills for IE. Eventually change other requests to use fetch as well. Future code deployment required to load getting started content from the back end instead of a js fetch.

> To test:
> - Open https://sdsmastersite.planmylegacy.org
> - Using devtools overrides:
>     - Override stelter-standard-v2-a0000000.css to exclude .ge-clt . You could also exlude crt and cga if you want to try hiding some additional sections.
>     - Override stelter.js with this pull request's stelter-source.js. 
>     - Reload the page. You might need to reload twice for the js override to properly take effect.
>     - Scroll down to "not sure how to begin" and click on "answer a few questions"
>         - Select the first radio option. You should only see 3 options because the fourth option is now hidden (it's suggestion is CLT, which has been excluded).


2. Fix for DMP sites switched over to digital guides that use global.js or global-v2.js. The form values were not being passed to the digital guides, so the user had to fill out the form on the guide as well. The fix is to port over a version of the Stelter.StandardFormsSetSession function from stelter.js.

> To test:
> - Open https://stelterstaging.ingeniuxondemand.com/msu-planmygift/wills-and-living-trusts
> - Using devtools overrides:
>     - Override global.js with this pull request's global-source.js. 
>     - Reload the page. You might need to reload twice for the js override to properly take effect.
>     - Fill out the Wills brochure form. You should be taken to the digital guide and not be required to fill out the form again.
